.gitignore

```diff
@@ -1,3 +1,5 @@
+.envrc
```

Comment:
just curious: what is this?

Reply:
See #71

Reply:
https://direnv.net/
I use it for activating the virtualenv upon entering the root directory of the project.

```
$ cat .envrc

source .venv/bin/activate
```
 you can also use it to load ENV variables (12-factor apps) etc.

Reply:
OK but this won't work with poetry out of the box (the virtualenv is somewhere in ~/.cache)

You could use `poetry shell`  or `source $(poetry env info -p)/bin/activate ` to achieve the same effect

And in this case, you won't need the `.venv` in the `.gitignore` at all.